sensors/nxt_i2c_sensor_defs.c

```diff
@@ -1103,12 +1103,12 @@ const struct nxt_i2c_sensor_info nxt_i2c_sensor_defs[] = {
 				.raw_max = 180,
```

Comment:
also, can't see it in the diff, but there is a documentation comment line above for value0 that needs to be changed to (0 to 359)